spec/controllers/idv/address_controller_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+
+RSpec.describe Idv::AddressController do
+  include IdvHelper
+
+  let(:user) { create(:user) }
+
+  let(:idv_session) { subject.idv_session }
+
+  let(:pii_from_doc) { Idp::Constants::MOCK_IDV_APPLICANT_WITH_SSN.stringify_keys }
+
+  let(:flow_session) do
+    {
+      'pii_from_doc' => pii_from_doc,
+    }
+  end
+
+  before do
+    stub_sign_in(user)
+    stub_analytics
+    stub_idv_steps_before_verify_step(user)
+    subject.idv_session.flow_path = 'standard'
+    subject.user_session['idv/doc_auth'] = flow_session
+  end
+
+  describe '#new' do
+    before do
+      get :new
+    end
+
+    it 'logs an analytics event' do
+      expect(@analytics).to have_logged_event('IdV: address visited')
+    end
+  end
+
+  describe '#update' do
```

Comment:
Recommend adding a spec for update logging an analytics event too. (Thank you for adding a controller spec! Maybe we can delete some things out of the feature spec?)